k8s/helm/templates/dogu-debug-mode-role.yaml

```diff
@@ -0,0 +1,17 @@
+# This handles all permissions necessary to enable/disable debug mode in admin dogu
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "k8s-ces-control.name" . }}-dogu-debug-mode-role
+  labels:
+    {{- include "k8s-ces-control.labels" . | nindent 4 }}
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["list"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["update", "create", "get", "list", "delete"]
+  - apiGroups: ["apps"]
+    resources: ["deployments", "deployments/scale"]
+    verbs: ["watch", "update", "list", "get"]
```

Comment:
This is a totally valid role definition. Anyhow the lot of brackets and double quotes makes it hard to understand which permissions are exactly here at the first glance.

From my perspective this equivalent is faster browsable even though it is longer (this suggestion counts for all roles):

```yaml
rules:
  - apiGroups:
    - ""
    resources: 
    - pods
    verbs: 
    - list
  - apiGroups:
    - ""
    resources: 
    - configmaps
    verbs: 
      - update
      - create" 
      - get
      - list
      - delete
  - apiGroups:
    - apps
    resources: 
      - deployments
      - deployments/scale
    verbs: 
      - watch
      - update
      - list
      - get
```

What do you think?